src/services/email.services.js

```diff
@@ -0,0 +1,57 @@
+const configOptions = require('nodemailer').createTransport({
+  service: 'gmail',
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: true,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+const sendActivationLink = async (to, link) => {
+  const transporter = await configOptions.sendMail({
```

Comment:
Consider renaming `configOptions` to `transporter` to reflect that this is the instance used to send emails. This will improve code readability and understanding.